src/index.html

```diff
@@ -182,37 +199,13 @@ <h1 class="settings-sub-heading">Features</h1>
         </div>
         <p>Sorry, wasn't ready…</p>
       </div>
+      <link rel="preload" as="script" href="js/page.js" onload="appendScript(href)">
```

Comment:
Can you elaborate a bit why you move this `link` here and not in `head`?

I'm sure there's a reason.

Reply:
To ensure the script doesn't load before the DOM is present

Reply:
Is it better than regular `link preload` in head and then `script defer` before body closing tag? I assumed it would unnecessry trigger DOM recalculations, while the old approach theoretically could avoid that. Was it wrong to assume?
Is this done to achieve a higher priority at a cost of a later request?

Reply:
`script defer` delays the browser loading spinner, which is what I'm trying to avoid.